rainmaker_orchestrator/tests/test_kimi_client.py

```
@@ -0,0 +1,457 @@
+        """Should handle None mode parameter."""
+        mock_openai_client = Mock(spec=OpenAI)
+        with patch('rainmaker_orchestrator.clients.kimi.OpenAI', return_value=mock_openai_client):
+            client = KimiClient()
+        
+        mock_response = Mock()
+        mock_response.choices = [Mock()]
+        mock_response.choices[0].message.content = 'Response'
+        mock_openai_client.chat.completions.create.return_value = mock_response
+        
+        # Should default to general mode (temperature 0.7)
+        result = client.generate('Test', mode=None)
```

Ruff (0.14.10)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)